### DISCUSSION TITLE
1mp video quality on hero 10

### DISCUSSION THREAD

**Discussion by fborille** [gopro/labs#1022] [Feature Requests]
Hey guys

i’m new here, is the a way to record 1mp videos on a hero 10? Any mods ?

thanks

fabio

**Comment by dnewman-gpsw** [comment]
1 mp - megapixel?   The camera doesn't have 720p.  Lowest res is 1080p (about 2MP.)
If you ment bit rate (Mb) you can set BITR to 1 for low quality video.
